crates/ruff_python_codegen/src/generator.rs

```diff
@@ -63,13 +64,20 @@ mod precedence {
     pub(crate) const MAX: u8 = 63;
 }
 
+#[derive(Default)]
+pub enum UnparseMode {
+    #[default]
+    Default,
+    Ast,
+}
```

Comment:
Nit: I think just `Mode` is enough. I'd rename `Ast` to `AstUnparse` and document that the intent is to emit the same output as Python's `ast.unparse` method

Reply:
I suggest documenting it at the variant level